<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buyer_NewPurchaseLog</class>
 <widget class="QDialog" name="Buyer_NewPurchaseLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="goods_num">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="goods_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="goods_id">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="place">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="producer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="storage_num">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="affirm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货品名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_goodsID">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货品编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_storageNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>进货数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_place">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>存放地：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_producer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>生产商：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_producer_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前库存：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
